tools/kubequery/kubequery-fleet.yml

```diff
@@ -0,0 +1,184 @@
+# Copyright (c) 2020-present, The kubequery authors
+# Copyright (c) 2022-present, Fleet
+#
+# This source code is licensed as defined by the LICENSE file found in the
+# root directory of this source tree.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR GPL-2.0-only)
+
+# Adapted from original provided by Kubequery project.
+
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubequery
+  labels:
+    app.kubernetes.io/name: kubequery
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kubequery-sa
+  namespace: kubequery
+  labels:
+    app.kubernetes.io/name: kubequery-sa
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kubequery-clusterrole
+  labels:
+    app.kubernetes.io/name: kubequery-clusterrole
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+rules:
+- apiGroups: ["", "admissionregistration.k8s.io", "apps", "autoscaling", "batch", "events.k8s.io", "networking.k8s.io", "policy", "rbac.authorization.k8s.io", "storage.k8s.io"]
+  resources: ["*"]
+  verbs: ["get", "list", "watch"]
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kubequery-clusterrolebinding
+  labels:
+    app.kubernetes.io/name: kubequery-clusterrolebinding
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+roleRef:
+  kind: ClusterRole
+  name: kubequery-clusterrole
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+- kind: ServiceAccount
+  name: kubequery-sa
+  namespace: kubequery
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubequery-config
+  namespace: kubequery
+  labels:
+    app.kubernetes.io/name: kubequery-config
+    app.kubernetes.io/part-of: kubequery
+    app.kubernetes.io/version: latest
+data:
+  enroll.secret: Y3nHXcZUBcFJc/7e6V6k1z7RG22rrAnQ
+  kubequery.flags: |
```

Comment:
Looking at the example in [`kubequery.flags.tls`](https://github.com/Uptycs/kubequery/blob/d2db2ab171308460d1677024b8943a6f9fa16cf0/etc/kubequery.flags.tls), it seems like we should be using paths in `/opt/uptycs/etc` instead of `/opt/uptycs/config`. The [`entrypoint.sh`](https://github.com/Uptycs/kubequery/blob/19218c65285285c2ea25158d11df8a9ea30f7c7e/bin/entrypoint.sh#L14) copies from `/opt/uptycs/config` to `/opt/uptycs/etc` so I'm not really sure if it makes a big difference.